### ISSUE TITLE
Final loss value

### ISSUE THREAD

**Issue by TopTea1** [Megvii-BaseDetection/DynamicRouting#31]
Hi @yanwei-li, first thanks for this great work ! I use this code for segmentation training on CITYSCAPES. I'm using the Dynamic-Raw 16 layers with ImageNet and after  60K steps the loss is already between 0.1 and 0.06, and seems to hardly decrease. So is it normal or is it decreasing too fast ? What should be the final loss after 186K iters ?

Thanks :)

**Comment by yanwei-li** [comment]
Hi, thanks for your interest in our work! It's normal that the loss is decreased quickly. In my experiments, the loss is stable at 0.11@60K and 0.06@186K with ImageNet Pretraining.

**Comment by TopTea1** [comment]
Thanks for your answer !
